ocs_ci/helpers/helpers.py

```diff
@@ -1729,6 +1729,57 @@ def verify_volume_deleted_in_backend(
         return False
 
 
+def delete_volume_in_backend(interface, img_uuid, pool_name=None):
```

Comment:
if the pool is passed, why do we need the interface to be passed too?

Reply:
It is mostly for future use when the function will support cephfs to know how to delete.

Reply:
what I meant is that I think that instead of passing the interface, you can fetch the interface from the given pool

Reply:
Done.